pkgs/development/libraries/java/lombok/default.nix

```diff
@@ -0,0 +1,15 @@
+{stdenv, fetchurl}:
+
+stdenv.mkDerivation rec {
+  name = "lombok-1.16.8";
+  src = fetchurl {url = "https://projectlombok.org/downloads/${name}.jar"; sha256 = "0s7ak6gx1h04da2rdhvc0fk896cwqm2m7g3chqcjpsrkgfdv4cpy";};
```

Comment:
Typically, the desired format is
```
fetchurl {
  url = ...;
  sha256 = ...;
};
```
Doesn't matter that much, but maintaining consistency is nice.